### DISCUSSION TITLE
Twice the load in Linux.

### DISCUSSION THREAD

**Discussion by Organarh** [smplayer-dev/smplayer#413] [General]
SMPlayer Version: 22.2.0 (revision 10060) (64-bit)
Hello / Why, when watching a video from youtube on a laptop with Windows 10, 30 frames of video gives a load on the processor of 10%, 60 frames of 20%. When viewing with Manjaro Linux video 30 frames CPU load 20%? 60 frames 40% ??? Thanks.

**Comment by smplayer-dev** [comment]
Try changing the video output, in preferences -> general -> video.

**Reply by Organarh** [reply]
I changed the settings and not only there. In Linux,(intel) vr 9 does not work correctly.

**Reply by smplayer-dev** [reply]
Try enabling hardware decoding and/or increasing the threads for decoding in preferences -> performance.

**Reply by Organarh** [reply]
enable hardware decoding and/or increase the number of streams. Doesn't help. 
Video 30 frames (16-20%)
Primary video stream: 
Format: vp9, Selected codec: vp9.
Default audio track
Format: opus.
The same VLC-player video: 30 frames 6-8%
Video codec H264-MPEG-4 AVC (part 10) (avc1)
Audio Codec MPEG AAC Audio(mp4a).
SMPlayer  video-audio stream outputs incorrectly. What to do ?

**Comment by Organarh** [comment]
Solution:
Settings / Network (Use adaptive streams (resolution up to 4k)) "turn off".
Reload the player.
